src/libraries/Client/Microsoft.Agents.Client/ConfigurationChannelHost.cs

```diff
@@ -17,7 +17,7 @@ public class ConfigurationChannelHost : IChannelHost
         private readonly IServiceProvider _serviceProvider;
         private readonly IConnections _connections;
 
-        public ConfigurationChannelHost(IServiceProvider systemServiceProvider, IConnections connections, IConfiguration configuration, string configSection = "ChannelHost")
+        public ConfigurationChannelHost(IServiceProvider systemServiceProvider, IConnections connections, IConfiguration configuration, string defaultChannelName, string configSection = "ChannelHost")
```

Comment:
`defaultChannelFactory` instead?

Reply:
its a name though, not a factory. 
this turns into the HttpClient Name that is picked up from the Service Collection.